scripts/tpu-cleanup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -x
+set -u
+
+zones=("us-central1-b" "us-central2-b" "europe-west4-a")
+
+for zone in ${zones[@]}; do
+  region=${zone%-*}
+  cluster=xl-ml-test-$region
+  gcloud container clusters get-credentials $cluster --region $region --project xl-ml-test
+
+  # Don't delete any TPUs for pods that are currently running
+  pods_to_save=$(mktemp)
+  for namespace in default automated; do
+    for status in Running Pending; do
```

Comment:
How do we distinguish if the status of TPUs is actually running? I see you comment `Don't delete any TPUs for pods that are currently running` but here includes `Running`. What's the difference?

Reply:
Basically, we make a list of every Kubernetes pod that might still be using a TPU, and then exclude any TPU associated with those pods. We can map from TPU VM to Kubernetes pod easily, because the TPUs are named like `tpu-${POD_UID}`

Reply:
Clarified this comment.

Reply:
Thanks! LGTM!